### ISSUE TITLE
Warning and potential issues when saving while script is running

### ISSUE THREAD

**Issue by fmagin** [justfoxing/ghidra_bridge#8]
When attempting to save (`File->Save` or `Ctrl+s`) when the bridge is running Ghidra issues a warning that the program is currently being modified by a python script and any changes made by that script will be lost and potential future errors might occur. 

I think currently this is an inherent limitation of the design (as a script instead of maybe an extension). This might not actually break anything if the bridge isn't used to modify anything. Some temporary fix would be to fix the issue that the script can't be canceled and restarted without restarting Ghidra (due to the address not being freed), so it could just be canceled, program could be saved and restarted.

**Comment by justfoxing** [comment]
Yup, that's a thing. The next thing I was thinking of, after landing the interactive fixes, was writing a Ghidra plugin to provide controls for starting/stopping correctly.
